projects/datorama/akita-ng-forms-manager/src/lib/forms-manager.ts

```diff
@@ -195,7 +195,7 @@ export class AkitaNgFormsManager<FormsState = any> {
     control: AbstractControl,
     arrControlFactory: ArrayControlFactory | HashMap<ArrayControlFactory>
   ) {
-    if (Array.isArray(formValue)) {
+    if (Array.isArray(formValue) && formValue[0] instanceof FormArray) {
```

Comment:
What's the use-case here?

Reply:
That specific line didn't cause the error but I thought maybe add it anyways just in case. I can remove it, if you would like

Reply:
My question is what the use-case is for this fix?

Reply:
I get ```TypeError: Cannot use 'in' operator to search for 'category_ids' in undefined``` when ```category_ids``` is an array of values. Which is caused by line# 207 assuming that im passing a  FormArray.

```
contentForm = this.form.group({
    title: [null, Validators.required],
    category_ids: [null, Validators.required], // <--- value can be array or null
});

constructor(
    private form: FormBuilder,
    private formsManager: AkitaNgFormsManager<FormsState>,
) {
    this.formsManager.upsert('options', this.contentForm);
}
```